### PR TITLE
Add multicluster remote gateway install YAML for mesh command

### PR DESCRIPTION
**Pull request by ostromart**
This is used in the instructions for setting up remote clusters.

### CODE REVIEW

File: data/examples/multicluster/istio-gateway.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    enabled: false
+
+  policy:
+    enabled: false
+
+  telemetry:
+    enabled: false
+
+  security:
+    enabled: false
```
Comment (ayj) [inline data/examples/multicluster/istio-gateway.yaml]: Citadel is required for mTLS.

Comment (ostromart) [reply data/examples/multicluster/istio-gateway.yaml]: This was actually totally wrong due to too many tasks going on in parallel. 
PTAL at the updated values which should correspond to directions here:
https://preliminary.istio.io/docs/setup/install/multicluster/gateways/#deploy-the-istio-control-plane-in-each-cluster
and here:
https://preliminary.istio.io/docs/setup/install/multicluster/shared-gateways/#setup-cluster-1-primary

**Comment by ostromart** [comment]
@ayj could you ptal?

**Review by ayj — APPROVED** [review]

**Pull request merged**